src/platforms/github/release.go

```diff
@@ -77,7 +77,7 @@ func createGitHubTag(tagName string, releaseName string, hash string) *github.Ta
 
 func getGithubOwnerAndRepo(remoteURL string) (string, string, error) {
 	// Parse the GitHub owner and repository name from the remote URL
-	ownerRepoRegex := regexp.MustCompile(`github\.com[:/](.*)/(.*)\.git`)
+	ownerRepoRegex := regexp.MustCompile(`github\.com[:/](.*)\/(.*?)(?:\.git|$)`)
```

Comment:
something I didn't notice before but can we put a regex101 saved test link in the comment above this to help explain this and the things it is trying to match?